plugins/sqlfluff-templater-dbt/test/conftest.py

```diff
@@ -10,4 +11,4 @@ def dbt_flags():
     # some test code in core/dbt/parser/models.py, ModelParser. render_update().
     # We've seen occasional runtime errors from that code:
     # TypeError: cannot pickle '_thread.RLock' object
-    dbt.flags.USE_EXPERIMENTAL_PARSER = True
+    os.environ["DBT_USE_EXPERIMENTAL_PARSER"] = "True"
```

Comment:
Second attempt at the `TypeError: cannot pickle '_thread.RLock' object` fix I mentioned in Slack. I saw the error again in the build. Apparently the original code wasn't the right way to set the flag. I "tested" this change by temporarily adding some assertions in the templater, dbt, and osmosis code. The flag appeared to be set correctly in all cases. 🤞 that this fixes it "for real".